### PR TITLE
Large UI whitespace, formatting and imports refactor

### PR DESCRIPTION
**Pull request by kvakvs**
Affects many UI files.
Changes mostly to imports and whitespace (untabify).
Please review with :gear: option "Ignore whitespace"

### CODE REVIEW

File: TLM/TLM/UI/IncompatibleModsPanel.cs [hunk 1]
```diff
@@ -139,25 +139,16 @@ public void Initialize()
             blurEffect.size = new Vector2(resolution.x, resolution.y);
             blurEffect.absolutePosition = new Vector3(0, 0);
             blurEffect.SendToBack();
-            blurEffect.eventPositionChanged += OnBlurEffectPositionChange;
-            blurEffect.eventZOrderChanged += OnBlurEffectZOrderChange;
-            blurEffect.opacity = 0;
-            blurEffect.isVisible = true;
-            ValueAnimator.Animate("ModalEffect", delegate (float val) { blurEffect.opacity = val; }, new AnimatedFloat(0f, 1f, 0.7f, EasingType.CubicEaseOut));
+            if (blurEffect != null)
```
Comment (krzychu124) [inline TLM/TLM/UI/IncompatibleModsPanel.cs]: Removing those callbacks will cause that bug with z-order of incompatible mods panel and blurry background when Paradox login notification opens

Comment (kvakvs) [reply TLM/TLM/UI/IncompatibleModsPanel.cs]: Wait a sec, did i accidentally change the logic?
The intent was to change delegates to lambdas only, and some line breaks

Comment (krzychu124) [reply TLM/TLM/UI/IncompatibleModsPanel.cs]: As you can see delegates were removed ;)

Comment (kvakvs) [reply TLM/TLM/UI/IncompatibleModsPanel.cs]: `val => blurEffect.opacity = val` is lambda
`() => blurEffect.Hide()` is another lambda

Comment (krzychu124) [reply TLM/TLM/UI/IncompatibleModsPanel.cs]: missing
`blurEffect.eventPositionChanged += OnBlurEffectPositionChange;` 
`blurEffect.eventZOrderChanged += OnBlurEffectZOrderChange;`

File: TLM/TLM/UI/IncompatibleModsPanel.cs [hunk 2]
```diff
@@ -139,25 +139,16 @@ public void Initialize()
...
+            {
+                blurEffect.isVisible = true;
+                ValueAnimator.Animate("ModalEffect", delegate (float val) { blurEffect.opacity = val; }, new AnimatedFloat(0f, 1f, 0.7f, EasingType.CubicEaseOut));
+            }
 
             // Make sure modal dialog is in front of all other UI
             BringToFront();
         }
 
-        private void OnBlurEffectPositionChange(UIComponent component, Vector2 position) {
```
Comment (krzychu124) [inline TLM/TLM/UI/IncompatibleModsPanel.cs]: why is that removed?

File: TLM/TLM/UI/IncompatibleModsPanel.cs [hunk 3]
```diff
@@ -343,10 +334,13 @@ private void TryPopModal()
                 }
             }
 
-            if (blurEffect != null && UIView.ModalInputCount() == 0) {
-                blurEffect.eventPositionChanged -= OnBlurEffectPositionChange;
-                blurEffect.eventZOrderChanged -= OnBlurEffectZOrderChange;
```
Comment (krzychu124) [inline TLM/TLM/UI/IncompatibleModsPanel.cs]: And this... :)

Comment (kvakvs) [reply TLM/TLM/UI/IncompatibleModsPanel.cs]: Let me double check if i pulled in a wrong version or fat fingered somewhere, or something

Comment (krzychu124) [reply TLM/TLM/UI/IncompatibleModsPanel.cs]: `JB Rider` don't like delegate `-=` so maybe removed that automatically

File: TLM/TLM/UI/IncompatibleModsPanel.cs (hunk 2 above)
Comment (kvakvs) [reply TLM/TLM/UI/IncompatibleModsPanel.cs]: Wrong version. Reapplied changes to this file only

File: TLM/TLM/UI/IncompatibleModsPanel.cs (hunk 3 above)
Comment (kvakvs) [reply TLM/TLM/UI/IncompatibleModsPanel.cs]: Wrong version, reapplied the changes

Comment (kvakvs) [reply TLM/TLM/UI/IncompatibleModsPanel.cs]: I had master from 10.21, not the fixed 10.21.1, very sorry
It was wrong idea to touch that file anyway :)

Comment (krzychu124) [reply TLM/TLM/UI/IncompatibleModsPanel.cs]: No problem :)

File: TLM/TLM/Manager/Impl/LaneArrowManager.cs [hunk 4]
```diff
@@ -1,168 +1,171 @@
...
+﻿namespace TrafficManager.Manager.Impl {
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using ColossalFramework;
+    using CSUtil.Commons;
+    using Geometry.Impl;
+    using State;
+    using Traffic;
+    using static State.Flags;
```
Comment (aubergine10) [inline TLM/TLM/Manager/Impl/LaneArrowManager.cs]: Might just be me, but I actually find the fully qualified using statements (with full namespace) easier to read as it makes it more obvious which stuff is from `TrafficManager`...

File: TLM/TLM/UI/IncompatibleModsPanel.cs [hunk 5]
```diff
@@ -346,7 +346,11 @@ private void TryPopModal()
             if (blurEffect != null && UIView.ModalInputCount() == 0) {
                 blurEffect.eventPositionChanged -= OnBlurEffectPositionChange;
                 blurEffect.eventZOrderChanged -= OnBlurEffectZOrderChange;
-                ValueAnimator.Animate("ModalEffect", delegate (float val) { blurEffect.opacity = val; }, new AnimatedFloat(1f, 0f, 0.7f, EasingType.CubicEaseOut), delegate () { blurEffect.Hide(); });
+                ValueAnimator.Animate(
```
Comment (aubergine10) [inline TLM/TLM/UI/IncompatibleModsPanel.cs]: Another delegate issue?

Comment (kvakvs) [reply TLM/TLM/UI/IncompatibleModsPanel.cs]: Not an issue, delegates are gone and replaced with lambdas, this is intended and does not change the logic.

File: TLM/TLM/Manager/Impl/LaneArrowManager.cs (hunk 4 above)
Comment (kvakvs) [reply TLM/TLM/Manager/Impl/LaneArrowManager.cs]: This is StyleCop SA1200, either accept it or disable it, there is no middle ground. Doesn't bother me at all as this section is minimized 99% of the time.

**Comment by kvakvs** [comment]
Closing this. A much better one is coming (file by file)
